src/main/java/hallpointer/address/logic/commands/AddSessionCommand.java

```
@@ -49,6 +49,7 @@ public class AddSessionCommand extends Command {
     public AddSessionCommand(Session session, Set<Index> memberIndexes) {
         requireNonNull(session);
         requireNonNull(memberIndexes);
+        assert !memberIndexes.isEmpty();
```

Should we throw an error instead for defensive coding?